acceptance_tests/features/steps/send_message_from_todo_list.py

```diff
@@ -0,0 +1,113 @@
+from behave import given, when, then
+
+from acceptance_tests import browser
+from acceptance_tests.features.pages import surveys_todo
+
+
+@given('the respondent is on their todo list')
+@when('the respondent navigates to their surveys todo list')
+def surveys_todo_list(context):
+    surveys_todo.go_to()
+
+
+@then('the respondent is able to send a message')
+def able_to_send_message_for_survey_ru(context):
+    browser.find_by_id('create-message-link-1')
+
+
+@given('the respondent chooses to send a message to ONS')
+@given('the respondent is sending a message in relation to bricks')
+@when('the respondent chooses to send a message for a specific RU and survey')
+def select_create_message(context):
+    surveys_todo_list(context)
+    browser.find_by_id('create-message-link-1').click()
+
+
+@then('the respondent is navigated to the create message page')
+def assert_create_message_page(context):
+    browser.find_by_id('secure-message-subject')
+
+
+@when('the respondent enters a valid message')
+def enter_valid_message(context):
+    browser.driver.find_element_by_id('secure-message-subject').send_keys('123')
+    browser.driver.find_element_by_id('secure-message-body').send_keys('345')
+
+
+@then('the message will be sent to the internal Bricks mailbox')
+def message_sent_to_bricks_workgroup(context):
+    current_url = browser.driver.current_url
+    assert 'survey=cb8accda-6118-4d3b-85a3-149e28960c54' in current_url
+
+
+@when('the respondent enters more than 96 characters in the subject field')
+def subject_field_too_long(context):
+    browser.driver.find_element_by_id('secure-message-subject').send_keys('x' * 98)
+    browser.driver.find_element_by_id('secure-message-body').send_keys('y' * 200)
+
+
+@when('chooses to send the message')
+@when('the message is sent')
+@when('selects to send message')
+def click_send_message_btn(context):
+    browser.find_by_id('send-message-btn').click()
+
+
+@then('an error message appears specifying subject too long')
+def check_for_error_subject_too_long(context):
+    browser.driver.find_element_by_link_text('Subject field length must not be greater than 100')
+
+
+@when('the respondent enters more than 10000 characters in the body field')
+def body_too_long(context):
+    browser.driver.find_element_by_id('secure-message-subject').send_keys('x' * 50)
+    browser.driver.find_element_by_id('secure-message-body').send_keys('y' * 11000)
```

Comment:
This step is extremely slow to run, you can achieve the same action much faster by using javascript like this:
```python
browser.execute_script(f'document.getElementById("secure-message-body").value="{"x" * 9990}";')
```
To enter the majority of the characters, then send the last few characters with send keys.
I had exactly the same issue on the internal user side.